components/micro-gateway-core/src/main/java/org/wso2/micro/gateway/core/validation/Validate.java

```diff
@@ -70,7 +71,12 @@ public static String validateRequest(String requestPath, String reqMethod, Strin
             return VALIDATED_STATUS;
         }
         String schema = extractSchemaFromRequest(requestPath, reqMethod, swagger);
-        return validateContent(payload, schema);
+        if (schema != null && !EMPTY_ARRAY.equals(schema) &&
```

Comment:
If schema is available and payload is null then, this will gives the status as true

Reply:
fixed.